### ISSUE TITLE
Cannot overfit model with no CRF option

### ISSUE THREAD

**Issue by dungtn** [glample/tagger#16]
Hi @glample,

I'm trying to overfit the LSTM without the CRF layer on a small dataset but the dev/test scores was about ~2%. I used the --crf 0 option, do I need to take any further steps?

**Comment by chqiwang** [comment]
What's your learning rate? Try use adadelta instead of sgd.

**Comment by dungtn** [comment]
It's because I didn't use the pre-trained skip100.
